### ISSUE TITLE
Button component using incorrect version member from platform object

### ISSUE THREAD

**Issue by patsissons** [react-native-training/react-native-elements#1209]
### Description

1. Explain what you did

Use the `Button` component with android version `>= 21`

2. What you expected to happen

I expect the background to use the ripple effect

3. What actually happens

The background uses the fallback selectable effect.

---

using `react-native-elements@1.0.0-beta5` the `Button` component is checking `Platform.VERSION` and should be checking `Platform.Version` instead (as per the [`react-native` source](https://github.com/facebook/react-native/blob/master/Libraries/Utilities/Platform.android.js#L17))

### Reproduction Steps and Sample Code

any usage of `<Button />` in android version `21` or newer.

### Solution

I will submit a PR shortly. In the mean time, sideload the version with `Platform.VERSION = Platform.Version`.

**Comment by patsissons** [comment]
a more complete augmentation solution for typescript:
```ts
import { Platform } from 'react-native';
declare module 'react-native' {
  export interface PlatformStatic {
    // this is a temp patch for react-native-element's button not rendering properly on android
    // see: https://github.com/react-native-training/react-native-elements/issues/1209
    VERSION: typeof Platform.Version;
  }
}
Platform.VERSION = Platform.Version;
```

**Comment by iRoachie** [comment]
Wow, not sure how we missed that the whole time 😆 thanks man

**Comment by patsissons** [comment]
no problemo 👍

**Comment by iRoachie** [comment]
Fixed now in [1.0.0-beta6](https://github.com/react-native-training/react-native-elements/releases/tag/v1.0.0-beta6)!
